components/validator/builtin_validators.rst

```diff
@@ -0,0 +1,8 @@
+.. index::
+    single: Validator; Built In validators
+
+Built In validators

```

Comment:
It should be Validators (and below as well). In the docs, we use this title case standard: Capitialize every word except from closed-class words. "Validators/Validation" are not closed class words and have to be capitialized.